Tested building all of the modules.

The `moreheaders` doesn't seem to work for both stable and mainline nginx versions though:

```
objs/addon/src/ngx_http_headers_more_filter_module.o:(.bss+0x0): multiple definition of `ngx_http_headers_more_location_hash'
objs/addon/src/ngx_http_headers_more_filter_module.o:(.bss+0x0): first defined here
objs/addon/src/ngx_http_headers_more_filter_module.o:(.data.rel.local+0x0): multiple definition of `ngx_http_headers_more_filter_module'
objs/addon/src/ngx_http_headers_more_filter_module.o:(.data.rel.local+0x0): first defined here
objs/addon/src/ngx_http_headers_more_headers_out.o: In function `ngx_http_headers_more_exec_cmd':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_out.c:112: multiple definition of `ngx_http_headers_more_exec_cmd'
objs/addon/src/ngx_http_headers_more_headers_out.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_out.c:112: first defined here
objs/addon/src/ngx_http_headers_more_headers_out.o: In function `ngx_http_headers_more_set_headers':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_out.c:500: multiple definition of `ngx_http_headers_more_set_headers'
objs/addon/src/ngx_http_headers_more_headers_out.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_out.c:500: first defined here
objs/addon/src/ngx_http_headers_more_headers_out.o: In function `ngx_http_headers_more_clear_headers':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_out.c:509: multiple definition of `ngx_http_headers_more_clear_headers'
objs/addon/src/ngx_http_headers_more_headers_out.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_out.c:509: first defined here
objs/addon/src/ngx_http_headers_more_headers_in.o: In function `ngx_http_headers_more_exec_input_cmd':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_in.c:172: multiple definition of `ngx_http_headers_more_exec_input_cmd'
objs/addon/src/ngx_http_headers_more_headers_in.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_in.c:172: first defined here
objs/addon/src/ngx_http_headers_more_headers_in.o: In function `ngx_http_headers_more_set_input_headers':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_in.c:462: multiple definition of `ngx_http_headers_more_set_input_headers'
objs/addon/src/ngx_http_headers_more_headers_in.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_in.c:462: first defined here
objs/addon/src/ngx_http_headers_more_headers_in.o: In function `ngx_http_headers_more_clear_input_headers':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_in.c:471: multiple definition of `ngx_http_headers_more_clear_input_headers'
objs/addon/src/ngx_http_headers_more_headers_in.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_headers_in.c:471: first defined here
objs/addon/src/ngx_http_headers_more_util.o: In function `ngx_http_headers_more_parse_header':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:22: multiple definition of `ngx_http_headers_more_parse_header'
objs/addon/src/ngx_http_headers_more_util.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:22: first defined here
objs/addon/src/ngx_http_headers_more_util.o: In function `ngx_http_headers_more_parse_statuses':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:164: multiple definition of `ngx_http_headers_more_parse_statuses'
objs/addon/src/ngx_http_headers_more_util.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:164: first defined here
objs/addon/src/ngx_http_headers_more_util.o: In function `ngx_http_headers_more_parse_types':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:229: multiple definition of `ngx_http_headers_more_parse_types'
objs/addon/src/ngx_http_headers_more_util.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:229: first defined here
objs/addon/src/ngx_http_headers_more_util.o: In function `ngx_http_headers_more_rm_header_helper':
/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:268: multiple definition of `ngx_http_headers_more_rm_header_helper'
objs/addon/src/ngx_http_headers_more_util.o:/nix/store/jsqrk045m09i136mgcfjfai8i05nq14c-source/src/ngx_http_headers_more_util.c:268: first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [objs/Makefile:396: objs/nginx] Error 1
make[1]: Leaving directory '/build/nginx-1.13.12'
make: *** [Makefile:8: build] Error 2
builder for '/nix/store/23q4dyyjhi927ahd2plr972vq2flaylp-nginx-1.13.12.drv' failed with exit code 2
```

--- a/pkgs/servers/http/nginx/modules.nix
+++ b/pkgs/servers/http/nginx/modules.nix
@@ -48,8 +48,8 @@
     src = fetchFromGitHub {
       owner = "openresty";
       repo = "headers-more-nginx-module";
-      rev = "v0.26";
-      sha256 = "0zhr3ai4xf5yghxvlbrwv8n06fgx33f1n1d4a6gmsczdfjzf8g6g";
+      rev = "v0.33";
+      sha256 = "1cgdjylrdd69vlkwwmn018hrglzjwd83nqva1hrapgcfw12f7j53";
     };
   };
 

--- a/pkgs/servers/http/nginx/modules.nix
+++ b/pkgs/servers/http/nginx/modules.nix
@@ -93,8 +93,8 @@
     src = fetchFromGitHub {
       owner = "openresty";
       repo = "lua-nginx-module";
-      rev = "v0.10.10";
-      sha256 = "1dlqnlkpn3pnhk2m09jdx3iw3m6xk31pw2m5xrpcmqk3bll68mw6";
+      rev = "v0.10.13";
+      sha256 = "19mpc76lfhyyvkfs2n08b4rc9cf2v7rm8fskkf60hsdcf6qna822";
     };
     inputs = [ pkgs.luajit ];
     preConfigure = ''

--- a/pkgs/servers/http/nginx/modules.nix
+++ b/pkgs/servers/http/nginx/modules.nix
@@ -19,9 +19,10 @@
     src = fetchFromGitHub {
       owner = "masonicboom";
       repo = "ipscrub";
-      rev = "99230f66d5afe1f929cf4ed217901acb6206f620";
-      sha256 = "0mfrwkg4srql38w713pg6qxi0h4hgy8inkvgc9cm80bwlv2ng9s1";
+      rev = "v1.0.1";
+      sha256 = "0qcx15c8wbsmyz2hkmyy5yd7qn1n84kx9amaxnfxkpqi05vzm1zz";
     } + "/ipscrub";
+    inputs = [ pkgs.libbsd ];
   };
 
   rtmp ={

